### PR TITLE
Added latest keyword in ls command.

### PR DESCRIPTION
**Pull request by ulziibuyan**
The restore command has the special "latest" snapshot keyword. Why not have same for listing files and directories? :-)

### CODE REVIEW

File: src/cmds/restic/cmd_ls.go [hunk 1]
```diff
@@ -86,9 +88,20 @@ func runLs(gopts GlobalOptions, args []string) error {
...
 
-	id, err := restic.FindSnapshot(repo, args[0])
-	if err != nil {
-		return err
+	snapshotIDString := args[0]
+	var id restic.ID
+	var paths []string
+
+	if snapshotIDString == "latest" {
+		id, err = restic.FindLatestSnapshot(repo, paths, "")
```
Comment (fd0) [inline src/cmds/restic/cmd_ls.go]: Have a look at how `FindLatestSnapshot` is used in the restore command. There, you can specify the list of paths and a hostname. I think we should add that here, too, to make the user interface consistent.

**Review by fd0 — CHANGES_REQUESTED** [review]
Thanks for the PR, please add the options for paths and hostname.

**Comment by ulziibuyan** [comment]
Options added. Should I open issue for this PR? Actually, just read CONTRIBUTING.MD

**Comment by fd0** [comment]
It's not necessary to add an additional issue, the PR is enough for now.

**Review by fd0 — APPROVED** [review]

**Comment by fd0** [comment]
LGTM, thanks for your work!

**Pull request merged**